rice-namenode/source/ClientNamenodeProtocolImpl.cc

```diff
@@ -86,6 +85,22 @@ ClientNamenodeTranslator::ClientNamenodeTranslator(
 
 // ----------------------- RPC HANDLERS ----------------------------
 
+
+
+
+
+std::string ClientNamenodeTranslator::append(std::string input) {
```

Comment:
Assign an owner here with `TODO`?

Ditto for the remaining handlers

Reply:
These won't be implemented until a later sprint when we implement name node functionalities, so I figured we could assign "owners" to the handlers then.